graphql/src/main/java/com/linecorp/armeria/common/scalar/PathScalar.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.scalar;
```

Comment:
What's your plan for this package? What other classes do you plan to add?

I find graphql-java has `graphql.Scalars`. How about adding `MoreScalars` like we did in `MoreMeters` and add `MoreScalars.path()` or `MoreScalars.GraphQLPath` ?

Reply:
> What's your plan for this package? What other classes do you plan to add?

Yes, that's right. If Armeria supports classes such as `MultipartFile` in Spring, I am thinking of adding a suitable scalar type.

See https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/multipart/MultipartFile.html

> I find graphql-java has graphql.Scalars. How about adding MoreScalars like we did in MoreMeters and add MoreScalars.path() or MoreScalars.GraphQLPath ?

Good idea. 😎